database/seeds/DevSeeder.php

```diff
@@ -218,5 +223,17 @@ public function run(): void
         $hrClosedJob->job_applications()->saveMany(factory(JobApplication::class, 5))->create([
             'job_poster_id' => $hrClosedJob->id
         ]);
+
+        // Create relationship between applicants and skills.
+        for ($i=0; $i < 100; $i++) {
+            DB::table('applicants_skills')->updateOrInsert(
+                [
+                    'applicant_id' => Applicant::inRandomOrder()->get()->first()->id
+                ],
+                [
+                    'skill_id' => Skill::inRandomOrder()->get()->first()->id
+                ]
+            );
+        }
     }
```

Comment:
I think it might be faster and more useful to focus on specific applicants here. Maybe just add 5 skills to $applicantUser->applicant. Also, the loop as is could accidentally add the same skill to the same applicant multiple times, causing a DB error.